### PR TITLE
Added flow with postgresql.

### PR DESCRIPTION
**Pull request by ggsdc**
One test has been disabled for now

This partially covers issue #75

### CODE REVIEW

File: .github/workflows/run_tests_postgresql.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -U -r requirements-dev.txt
+    - name: Run unit tests
+      run: python -m unittest discover -s cornflow.tests.unit
+      env:
+        FLASK_ENV: testing
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: 5432
```
Comment (pchtsp) [inline .github/workflows/run_tests_postgresql.yml:45]: should these two be here? or in the postgres service configuration above?

Comment (ggsdc) [reply .github/workflows/run_tests_postgresql.yml:45]: In the examples given by GitHub they place those values there:

https://docs.github.com/en/actions/guides/creating-postgresql-service-containers#running-jobs-directly-on-the-runner-machine

File: cornflow/tests/unit/test_roles.py [hunk 2]
```diff
@@ -61,11 +61,11 @@ def test_get_no_roles(self):
                 )
                 self.assertEqual(403, response.status_code)
```
Comment (ggsdc) [inline cornflow/tests/unit/test_roles.py:63]: This is referenced in issue #77 , I think I already know why this is happening, but I want to try something out and leave the issue open with the idea of how to fix it so someone can do it.

File: .github/workflows/run_tests_postgresql.yml (hunk 1 above)
Comment (pchtsp) [reply .github/workflows/run_tests_postgresql.yml:45]: I checked the example: actually we do not need those env vars.

Check what they say in the comments:
```
# Runs a script that creates a PostgreSQL table, populates
# the table with data, and then retrieves the data
run: node client.js
# Environment variables used by the `client.js` script to create
# a new PostgreSQL table.
env:
  # The hostname used to communicate with the PostgreSQL service container
  POSTGRES_HOST: localhost
	  # The default PostgreSQL port
	  POSTGRES_PORT: 5432
```
So the environment variables are for the node client.js example.

Comment (ggsdc) [reply .github/workflows/run_tests_postgresql.yml]: I have deleted the environment variables and it works. I will perform the merge.

**Pull request merged**